Movies-Selection/Movies-Selection/Helpers/Builder/AssemblyModule.swift

```diff
@@ -15,16 +15,27 @@ final class AssemblyModule: AssemblyBuilderProtocol {
     func createMovieModule(router: RouterProtocol) -> UIViewController {
         let view = MovieTableViewController()
         let movieAPIService = MovieAPIService()
-        let presenter = MoviePresenter(view: view, APIService: movieAPIService, router: router)
-        presenter.getMovies(view: view)
+        let dataStorageService = MoviesRepository()
```

Comment:
```suggestion
        let dataStorageService = DataStorageService()
```
DataStorageService по тз!